packages/amplify-migration/src/commands/gen2/execute/execute_command.ts

```
@@ -42,28 +44,67 @@ export class Gen2ExecuteCommand implements CommandModule<object, ExecuteCommandO
         type: 'string',
         demandOption: true,
       })
-      .option('customResourceMap', {
-        describe: 'Path to the custom category resource map JSON file',
+      .option('resourceMappings', {
+        describe: `Path to the resource mappings JSON file. E.g file://complete/path/to/file. The JSON is an array of resource mappings
+        where each mapping consists of a Source object and Destination object with each of them containing the StackName and LogicalResourceId
+        of the resource to move. E.g:
+        [{
+          "Source": {
+            "StackName": "myStackA",
+            "LogicalResourceId": "myLogicalIdA"
+          },
+          "Destination": {
+            "StackName": "myStackB",
+            "LogicalResourceId": "myLogicalIdB"
+          }
+        }]`,
         type: 'string',
         demandOption: false,
       });
   };
   handler = async (args: ArgumentsCamelCase<ExecuteCommandOptions>): Promise<void> => {
-    const { from, to, customResourceMap } = args;
+    const { from, to, resourceMappings } = args;
     assert(from);
     assert(to);
 
-    let parsedcustomResourceMap: ResourceMapping[] | undefined = undefined;
+    let parsedResourceMappings: ResourceMapping[] | undefined = undefined;
 
-    if (customResourceMap) {
+    if (resourceMappings) {
+      if (!resourceMappings.startsWith(FILE_PROTOCOL_PREFIX)) {
+        throw new Error(`Expected resourceMap to start with ${FILE_PROTOCOL_PREFIX}`);
+      }
+      const resourceMapPath = resourceMappings.split(FILE_PROTOCOL_PREFIX)[1];
```

Should we check the length of resourceMappings after split? If length is 2, then we get [1]
If resourceMappings is only `file://` then this would fail right?

---

yup, that's a good validation to fail fast on :)